platformio.ini

```diff
@@ -165,18 +95,66 @@ lib_deps =
     ; https://github.com/adafruit/Adafruit_MAX1704X @ 1.0.2
   #For MPU6050 IMU uncomment follwoing
     ;electroniccats/MPU6050 @1.0.1
-  # For -D USERMOD_ANIMARTRIX
+  # For -DUSERMOD_ANIMARTRIX
   # CC BY-NC 3.0 licensed effects by Stefan Petrick, include this usermod only if you accept the terms!
     ;https://github.com/netmindz/animartrix.git#18bf17389e57c69f11bc8d04ebe1d215422c7fb7
   # SHT85
     ;robtillaart/SHT85@~0.3.3
   # Audioreactive usermod
     ;kosme/arduinoFFT @ 2.0.1
 
-extra_scripts = ${scripts_defaults.extra_scripts}
+extra_scripts =
+  pre:pio-scripts/set_version.py
+  post:pio-scripts/output_bins.py
+  post:pio-scripts/strip-floats.py
+  pre:pio-scripts/user_config_copy.py
+  pre:pio-scripts/build_ui.py
+  ; post:pio-scripts/obj-dump.py  ;; convenience script to create a disassembly dump of the firmware (hardcore debugging)
+
+# ------------------------------------------------------------------------------
+# Available lwIP variants (macros):
+#    -DPIO_FRAMEWORK_ARDUINO_LWIP_HIGHER_BANDWIDTH  = v1.4 Higher Bandwidth (default)
+#    -DPIO_FRAMEWORK_ARDUINO_LWIP2_LOW_MEMORY       = v2 Lower Memory
+#    -DPIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH = v2 Higher Bandwidth
+#    -DPIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH_LOW_FLASH
+#
+# BearSSL performance:
+#  When building with -DSECURE_CLIENT=SECURE_CLIENT_BEARSSL, please add `board_build.f_cpu = 160000000` to the environment configuration
+#
+# BearSSL ciphers:
+#   When building on core >= 2.5, you can add the build flag -DBEARSSL_SSL_BASIC in order to build BearSSL with a limited set of ciphers:
+#     TLS_RSA_WITH_AES_128_CBC_SHA256 / AES128-SHA256
+#     TLS_RSA_WITH_AES_256_CBC_SHA256 / AES256-SHA256
+#     TLS_RSA_WITH_AES_128_CBC_SHA / AES128-SHA
+#     TLS_RSA_WITH_AES_256_CBC_SHA / AES256-SHA
+#  This reduces the OTA size with ~45KB, so it's especially useful on low memory boards (512k/1m).
+# ------------------------------------------------------------------------------
+build_flags =
+  -std=gnu++17
```

Comment:
Why force gnu++17 ? 
I'd prefer we stay with what espressif has set in their platform files.

Reply:
I consider the c++ version a version discussion, too, so let's discuss this later.